tests/acceptance/bootstrap/AuthAppContext.php

```diff
@@ -0,0 +1,138 @@
+<?php declare(strict_types=1);
+/**
+ * ownCloud
+ *
+ * @author Niraj Acharya <niraj@jankaritech.com>
+ * @copyright Copyright (c) 2024 Niraj Acharya niraj@jankaritech.com
+ *
+ * This code is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License,
+ * as published by the Free Software Foundation;
+ * either version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>
+ *
+ */
+
+use Behat\Behat\Context\Context;
+use Behat\Behat\Hook\Scope\BeforeScenarioScope;
+use TestHelpers\BehatHelper;
+use GuzzleHttp\Exception\GuzzleException;
+use TestHelpers\AuthAppHelper;
+
+require_once 'bootstrap.php';
+
+/**
+ * AuthApp context
+ */
+class AuthAppContext implements Context {
+	private FeatureContext $featureContext;
+	private array $allCreatedTokens = [];
+
+	/**
+	 * @BeforeScenario
+	 *
+	 * @param BeforeScenarioScope $scope
+	 *
+	 * @return void
+	 */
+	public function before(BeforeScenarioScope $scope): void {
+		// Get the environment
+		$environment = $scope->getEnvironment();
+		// Get all the contexts you need in this context
+		$this->featureContext = BehatHelper::getContext($scope, $environment, 'FeatureContext');
+	}
+
+	/**
+	 * @When the administrator creates app token with expiration time :expiration using the API
+	 *
+	 * @param string $expiration
+	 *
+	 * @return void
+	 */
+	public function theAdministratorCreatesAppTokenForUserWithExpirationTimeUsingTheApi(string $expiration): void {
```

Comment:
Just matching with the step-def
```suggestion
	public function theAdministratorCreatesAppTokenWithExpirationTimeUsingTheApi(string $expiration): void {
```